pkg/cloudprovider/providers/vsphere/vsphere_util.go

```diff
@@ -72,7 +72,12 @@ func getVSphereConfig() (*VSphereConfig, error) {
 	if err != nil {
 		return nil, err
 	}
-	defer confFile.Close()
+	defer func() {
+		if err := confFile.Close(); err != nil {
```

Comment:
Can just ignore this error `_ := ...` since the file is only opened for reading.